docs/DeveloperGuide.adoc

```diff
@@ -16,24 +16,31 @@
 . Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
 . Set up the correct JDK version
 .. Click `Configure` > `Project Defaults` > `Project Structure`
-.. If JDK 8 is listed in the drop down, select it. If it is not, click `New...` and select the directory where you installed JDK 8.
+.. If JDK 9 is listed in the drop down, select it. If it is not, click `New...` and select the directory where you installed JDK 9.
 .. Click `OK`.
 . Click `Import Project`
 . Locate the project directory and click `OK`
 . Select `Create project from existing sources` and click `Next`
 . Rename the project if you want. Click `Next`
 . Ensure that your `\src` and `\test\java` folders are checked. Keep clicking `Next`
 . Click `Finish`
+. Try running the application (right-click the `Main` class under the `src` folder, and click `Run 'Main.main()'`)
+. Observe how the application fails to compile. This is because the `JAXB` APIs is no longer contained under the default classpath from Java 9 onwards. To fix this issue, the `JAXB` module needs to be added to the classpath:
```

Comment:
I played around with the codebase a little more.

Actually, we can fix the issue of the missing modules on our side, by adding a `module-info.java` to `src/`:

```java
open module addressbook.level2 {
    requires java.xml.bind;
    exports seedu.addressbook.storage.jaxb;
}
```

After doing that, students need not add `--add-modules` manually as specified in the instructions anymore.

This doesn't seem to work on tests though, not sure if `module-info.java` can be used on tests as well? Or perhaps it is placed wrongly?

Also not sure whether this entire solution is the best to begin with. What is your opinion on this?

Reply:
Based on this [issue](https://youtrack.jetbrains.com/issue/IDEA-171419) with Intellij, seems like the tests are run on the classpath instead of the modulepath, so it is unable to get the modules in `module-info.java`.

Reply:
> Although, there will still be a need to specify the --add-modules java.xml.bind flag (or the module path) in the command line and for the tests.

Yeah, I don't think there's a way around that.